server/src/main/java/org/opensearch/cluster/metadata/MetadataUpdateSettingsService.java

```diff
@@ -127,7 +130,10 @@ public void updateSettings(
             .normalizePrefix(IndexMetadata.INDEX_SETTING_PREFIX)
             .build();
 
-        MetadataCreateIndexService.validateRefreshIntervalSettings(normalizedSettings, clusterService.getClusterSettings());
+        validateRefreshIntervalSettings(normalizedSettings, clusterService.getClusterSettings());
+        if (isRemoteStoreAttributePresent(clusterService.getSettings())) {
```

Comment:
should we move this check to `validateTranslogDurabilitySettings` itself ? That way all the checks are contained there.

Reply:
Done